packages/quantic/decisions/0001-testing-strategy.md

```diff
@@ -39,12 +39,15 @@ Key points for Playwright E2E tests:
 - **User Workflow Testing**: Focus on the essential paths users take to accomplish their goals with a given component, this for all the different Quantic use cases: Search, Insight, Case assist and Recomendations.
 - **Simulate User Actions**: Test realistic user actions, such as clicking buttons, entering data, and navigating through the UI.
 - **External System Interactions**: Verify interactions with external systems, like Coveo APIs and analytics, to ensure components behave as expected in a real-world environment. This also acts as a double test because it will also notify us by failing the test if the external API contract is broken too because we test for the expected request body but also the expected response as much as possible.
+- **Browser-specific behaviors**: Test browser-specific behaviors that would interact with our components.
```

Comment:
we agreed with etienne that everything that is impacted by browser-specific behaviors should probably be tested in the e2e tests. I added some examples, but for example, changing the URL param could impact which tab is active and selected. stuff like that

Reply:
I agree that changes in the URL should be E2E tested, but calling Browser Specific Behaviour  is a bit miss leading as personally it will not make me  think about urls.

Reply:
yeah its to include anything browser related not just urls